content/en/blog/argo-insurance-workflow/index.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "Fission Function Orchestration with Argo Workflows"
+date = "2022-09-21T10:15:00+05:30"
+author = "Neha Gupta"
+description = "Multiple Fission function orchestration using Argo Workflows"
+categories = ["Tutorials"]
+type = "blog"
+images = ["images/featured/fission-zapier-featured.png"]
++++
+
+Fission Functions allow user to perform one logical task. To group multiple task together such as one function is dependent on other we can use Argo Workflows.
+Argo Workflows is an open source container-native workflow engine with a feature to create DAGs i.e. running task sequentially, in parallel and with dependencies. We will try to develop a simple Insurance Eligibility program which will take different input and calculate insurance installment on basis of the inputs.
+
+### Understanding the Insurance Calculator
+
+The small application/Workflow will take basic information from the user and check the eligibility for the Insurance. Below are the input and different tasks which are performed to calculating the above:
```

Comment:
Minor changes:
- This application will take basic information... _(remove the word small)_
- ...check their eligibility for Insurance (_remove the before eligibility and insurance_)